website/src/logged_in/logs/Logs.js

```diff
@@ -84,7 +225,22 @@ const columns = [
         width: 150,
         valueGetter: (params) =>
             `${params.getValue('prediction_results')['energy_consumption_kwh'].split(".")[0] || ''}`,
-    }
+    },
+    {
+        field: 'totalCost',
+        headerName: 'Total Cost (USD)',
+        description: 'Total Cost (USD)',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['total_cost']}`
```

Comment:
While this will be needed, the current system does not support pricing. I will add that in a different PR. Please remove this column from your PR.